concrete/blocks/image/view.css

```diff
@@ -0,0 +1,9 @@
+img.ccm-image-center {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+img.ccm-image-right {
+    margin-left: auto;
+    margin-right: 0;
+}
```

Comment:
These styles will both need `display: block;` (by default img elements are inline). I don't believe you need `img` combined with the class selector.